.github/workflows/scripts/comments.mjs

```diff
@@ -157,6 +264,12 @@ function githubActionRunUrl(context) {
  * @property {string} markerText - The marker to identify the comment
  */
 
+/**
+ * @typedef {object} StatComment
+ * @property {string} titleText - The title of the comment
+ * @property {string} markerText - The marker to identify the comment
+ */
```

Comment:
Not a blocker, but we could share (or rename) the `DiffComment` type

The workspace path could satisfy `path` to stop any warnings

```mjs
await commentStats({ github, context, commit }, issueNumber, {
  path: '${{ github.workspace }}',
  titleText: ':clipboard: Stats',
  markerText: 'stats'
})
```